### PR TITLE
Adds a few more examples and extra markdown.

### PR DESCRIPTION
**Pull request by matthewkapfhammer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,44 +14,77 @@ Support for:
...
 fs = fileseq.FrameSet("1-5")
 for f in fs:
   print f
 ```
 
-Random Access
--------------
+### Access Frames
 
+#### Using Indecies:
```
Comment (justinfx) [inline README.md]: `s/Indecies/Indices/`

File: README.md [hunk 2]
```diff
@@ -16,44 +14,77 @@ Support for:
...
 ```
 
-Finding Sequences on Disk
-=========================
+### Format Path for VFX Software
 
+```python
+>>> seq = fileseq.FileSequence("/foo/bar.1-10#.exr")
+>>> ''.join([seq.directory(), seq.basename(), seq.padding(), seq.extension()]) 
+"/foo/bar.#.exr"
```
Comment (justinfx) [inline README.md]: Maybe if you are showing an example of formatting the paths, you should also include an example of using the `FileSequence.format()` method?

File: README.md (hunk 1 above)
Comment (matthewkapfhammer) [inline README.md]: Haha. Cheers for that.

**Comment by matthewkapfhammer** [comment]
Changes are up.

Is there a reason why FileSequence.format()'s default template doesn't include the dirname?

**Comment by justinfx** [comment]
Hmm, I don't know of the original reason that it doest include the dirname
by default. Chambers could comment on that.

**Pull request closed without merging**